File: WarehouseApp/Models/DailySummary.swift
import Foundation

struct DailySummary: Identifiable, Codable {
    var id: UUID = UUID()
    let date: Date
    let totalVentas: Double
    let totalMP: Double
    let totalEfectivo: Double
    let cantidadVentas: Int
    var operadorName: String?

    var dominantMethod: String {
        totalMP >= totalEfectivo ? "QR / MP" : "Efectivo"
    }
}

File: LektaAndroid/app/src/main/java/com/lekta/app/models/DailySummary.kt
package com.lekta.app.models

import java.util.UUID

data class DailySummary(
    val id: String = UUID.randomUUID().toString(),
    val date: Long = System.currentTimeMillis(),
    val totalVentas: Double,
    val totalMP: Double,
    val totalEfectivo: Double,
    val cantidadVentas: Int,
    val operadorName: String? = null
) {
    val dominantMethod: String
        get() = if (totalMP >= totalEfectivo) "QR / MP" else "Efectivo"
}
